docs/pages/getting-started/linux-server.mdx

```diff
@@ -166,12 +166,26 @@ On your Linux machine, run the following command to start the `teleport` daemon
   </TabItem>
 
   <TabItem label="Source or custom install">
+  1. Create a systemd unit file using the `teleport install systemd` command and the [unit file path](https://www.freedesktop.org/software/systemd/man/systemd.unit.html) you'd
+  like to use for your configuration. The `/etc/systemd/system/teleport.service` path is recommended in most use case:
+
+    ```code
+    $ sudo teleport install systemd -o /etc/systemd/system/teleport.service
+    ```
+  2. Enable and start the new teleport service daemon:
+
     ```code
-    $ sudo teleport start
+    $ sudo systemctl enable teleport && sudo systemctl start teleport
     ```
   </TabItem>
 </Tabs>
 
+<Admonition title="The Teleport daemon" type="tip">
+Configuration for the teleport daemon relies on [systemd](https://www.freedesktop.org/wiki/Software/systemd/). For more information on how the
+teleport service daemon is configured, see our guide on how to [Run Teleport as a Daemon](https://goteleport.com/docs/setup/admin/daemon/).
```

Comment:
```suggestion
teleport service daemon is configured, see our guide on how to [Run Teleport as a Daemon](../setup/admin/daemon.mdx).
```

Reply:
Using internal links is better when possible, we will verify them more efficiently.

Reply:
Good point, I completed this for all `goteleport.com` links